packages/toolpad-studio/src/toolpad/FunctionsEditor/index.tsx

```diff
@@ -389,12 +395,12 @@ export default function FunctionsEditor() {
 
                 {introspection.isLoading ? (
                   <React.Fragment>
-                    <TreeItem disabled nodeId="::loading::" label={<Skeleton />} />
-                    <TreeItem disabled nodeId="::loading::" label={<Skeleton />} />
-                    <TreeItem disabled nodeId="::loading::" label={<Skeleton />} />
+                    <TreeItem disabled itemId="::loading::" label={<Skeleton />} />
```

Comment:
You don't have any issues rendering 3 items with the same ID?
This should crash or at least cause bugs

Reply:
Looks like this code is part of an unreleased part of the ui behind a feature flag. It seems to crash indeed. I added unique ids.